src/sentry/api/endpoints/organization_events_vitals.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+import sentry_sdk
+
+from rest_framework.response import Response
+from rest_framework.exceptions import ParseError
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, NoProjects
+from sentry.api.event_search import get_function_alias
+from sentry.snuba import discover
+
+
+class OrganizationEventsVitalsEndpoint(OrganizationEventsV2EndpointBase):
+    VITALS = {
+        "measurements.lcp": {"thresholds": [0, 2500, 4000]},
```

Comment:
Are we hardcoding these to just web vitals? Do we need groups if that's the case? I highly doubt this will be extended beyond the 3 thresholds in the next year or two.

Reply:
When we get around to custom measures, we could have thresholds be configurable via settings. I think this will work for now, and could extend into backend measures when they get added.